ansible/templates/minigraph_cpg.j2

```diff
@@ -226,56 +241,28 @@
           </BGPPeer>
 {% endif %}
 {% endfor %}
+{% endif %}
         </a:Peers>
         <a:RouteMaps/>
       </a:BGPRouterDeclaration>
 {% endfor %}
+{% endif %}
 {% if switch_type is defined and (switch_type == 'voq' or switch_type == 'chassis-packet') %}
 {% for a_linecard in all_loopback4096 %}
 {% if a_linecard != inventory_hostname %}
 {% for idx in range(all_loopback4096[a_linecard]|length) %}
       <a:BGPRouterDeclaration>
         <a:ASN>{{ vm_topo_config['dut_asn'] }}</a:ASN>
+{% if switch_type == 'voq' %}
+        <a:Hostname>{{ chassis_ibgp_peers[all_inbands[a_linecard][idx].split('/')[0]] }}</a:Hostname>
+{% else %}
         <a:Hostname>{{ chassis_ibgp_peers[all_loopback4096[a_linecard][idx].split('/')[0]] }}</a:Hostname>
+{% endif %}
         <a:RouteMaps/>
       </a:BGPRouterDeclaration>
 {% endfor %}
 {% endif %}
 {% endfor %}
-{% if num_asics > 1 %}
-{% for asic_id in range(num_asics) %}
-{% set asic_name = "ASIC" + asic_id|string %}
-      <a:BGPRouterDeclaration>
-        <a:ASN>{{ vm_topo_config['dut_asn'] }}</a:ASN>
-        <a:Hostname>{{ asic_name }}</a:Hostname>
```

Comment:
why this is removed ?

Reply:
Now that we asic_topo_config defined for multi-asic line cards, this is covered by iterating over asic_topo_config at line 203.